### ISSUE TITLE
Building Hello Go - curl example incorrect

### ISSUE THREAD

**Issue by Xaeco** [geerlingguy/ansible-for-kubernetes#103]
Under **Building Hello Go** on page 23

The second curl example is incorrect
```
curl localhost:8180
Hello, you requested: /test
```
should be
```
curl localhost:8180/test
Hello, you requested: /test
```

**Comment by geerlingguy** [comment]
Marking that first issue as a bug—for the second, what syntax is that? If PowerShell, I explicitly don't _directly_ support PowerShell in all my examples. I often recommend WSL2 just because it would be incredibly painful to try to match all commands between Windows/PowerShell and [everything else].

**Comment by Xaeco** [comment]
Yeah it's Windows command line. My WSL2 was broken so I installed the Windows ver alongside my windows ver of Docker desktop and had this issue. I think using Minikube docker-env is quite nice as it outputs the right command for the user's OS. I guess it depends if it's worth it given this shouldn't happen in WSL2.
